lms/djangoapps/commerce/api/v1/serializers.py

```diff
@@ -0,0 +1,38 @@
+""" API v1 serializers. """
+from rest_framework import serializers
+
+from commerce.api.v1.models import Course
+from course_modes.models import CourseMode
+
+DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+
+class CourseModeSerializer(serializers.ModelSerializer):
+    """ CourseMode serializer. """
+    name = serializers.CharField(source='mode_slug')
+    price = serializers.IntegerField(source='min_price')
+    expires = serializers.DateTimeField(source='expiration_datetime', required=False, default=None,
```

Comment:
What does it mean for a `CourseMode` to expire? Is this akin to the "availability date" we discussed earlier today? A comment here might be nice.